test/src/common/harnessIo.c

```diff
@@ -0,0 +1,75 @@
+/***********************************************************************************************************************************
+Harness for Testing
```

Comment:
Would be good to have this as a better comment ...

Reply:
Updating to Harness for Io Testing would be in line with the others.